static/src/javascripts/projects/common/modules/commercial/dfp/breakout-iframe.js

```diff
@@ -22,58 +30,50 @@ define([
      */
     function breakoutIFrame(iFrame, $slot) {
         /*eslint-disable no-eval*/
-        var shouldRemoveIFrame = false,
-            $iFrame            = bonzo(iFrame),
-            iFrameBody         = iFrame.contentDocument.body,
-            $iFrameParent      = $iFrame.parent(),
-            type               = {};
+        var $iFrame            = bonzo(iFrame);
+        var $iFrameParent      = bonzo(iFrame.parentNode);
+        var iFrameBody         = iFrame.contentDocument.body;
+        var type               = {};
+        var $breakoutEl;
 
-        if (iFrameBody) {
-            forEach(breakoutClasses, function (breakoutClass) {
-                $('.' + breakoutClass, iFrameBody).each(function (breakoutEl) {
-                    var creativeConfig,
-                        $breakoutEl     = bonzo(breakoutEl),
-                        breakoutContent = $breakoutEl.html();
+        if (!iFrameBody) {
+            return type;
+        }
 
-                    if (breakoutClass === 'breakout__script') {
-                        // new way of passing data from DFP
-                        if ($breakoutEl.attr('type') === 'application/json') {
-                            creativeConfig = JSON.parse(breakoutContent);
-                            if (creativeConfig.name === 'fluid250-v4' || creativeConfig.name === 'fluid250-v3') {
-                                creativeConfig.name = 'fluid250';
-                            }
-                            require(['common/modules/commercial/creatives/' + creativeConfig.name], function (Creative) {
-                                new Creative($slot, creativeConfig.params, creativeConfig.opts).create();
-                            });
-                        } else {
-                            // evil, but we own the returning js snippet
-                            eval(breakoutContent);
+        $breakoutEl = $('.breakout__html', iFrameBody);
+        if ($breakoutEl.length) {
+            fastdom.write(function () {
+                $iFrame.hide();
+                $breakoutEl.detach();
+                $iFrameParent.append($breakoutEl.html());
```

Comment:
It's a corner case, but this may not work if a creative ever comes back with more than one element classed `.breakout-html`. Bonzo's .html() method only returns the innerHTML of the first element in a Bonzo collection. Would it be safer just to append the `breakoutEl` rather than the serialized markup? I guess it's a corner case.